### ISSUE TITLE
Mention `failedJobsHistoryLimit` and `successfulJobsHistoryLimit` in CronJob concept page

### ISSUE THREAD

**Issue by phiroict** [kubernetes/website#38771]
Does not describe the fields `failedJobsHistoryLimit` and `successfulJobsHistoryLimit`, please add a list of all fields the spec can have, preferable with examples of use and format.

**Comment by sftim** [comment]
/retitle Mention `failedJobsHistoryLimit` and `successfulJobsHistoryLimit` in CronJob concept page

https://k8s.io/docs/concepts/workloads/controllers/cron-jobs/ does link to https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/cron-job-v1/ which includes an explanation of these fields.

/priority backlog
/sig apps

**Comment by phiroict** [comment]
Hmm, does not make sense to have these fields described by having to follow global doc link at the footer of the document. The exhaustive list of fields should be shown in the main page and for more information clearly linked to a detail page if needed. For starting users this is not clear and I can imagine for somebody not in the known you can only find this through the StackOverflow route. It also now fails to explain the possibilities of the definition as a person not aware of these fields is not informed of them as they would not know to look for them.  Not a great Documentation Experience this way.

**Comment by sftim** [comment]
This issue is ready for a volunteer to work on it. Would anyone like to help out?

**Comment by sftim** [comment]
> Does not describe the fields `failedJobsHistoryLimit` and `successfulJobsHistoryLimit`

Let's add that

---

> Please add a list of all fields the spec can have, preferable with examples of use and format.

This issue is now **not** about listing _all_ the fields; for that, we already have https://kubernetes.io/docs/reference/kubernetes-api/workload-resources/cron-job-v1/ (and that's why I changed the title of the issue)

**Comment by mrgiles** [comment]
There is a brief description of those fields in the document: https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#jobs-history-limits

@sftim, are you asking for additional details? There isn't a lot more to say about them other than maybe rephrase it so it is more clear?

**Comment by sftim** [comment]
Rephrasing the page to be more clear would be a welcome improvement.

> The exhaustive list of fields should be shown in the main page and for more information clearly linked to a detail page if needed.

Since this was filed, we've changed the page to link to the API reference from the right hand nav. We also have a long standing issue for [fact sheets](https://github.com/kubernetes/website/issues/38682), so work on this issue does not need to add a fact sheet.

A brief tidy-up of the job history limit explanation would be fine to close this issue. @mrgiles if you're happy with the wording as-is, you are welcome to close the issue already.

**Comment by mrgiles** [comment]
Thanks @sftim, see if you like the wording in the PR
